### DISCUSSION TITLE
Possibility of playing with annotations

### DISCUSSION THREAD

**Discussion by dougieled** [hrynko/vue-pdf-embed#76] [Ideas]
Hi, Just wondering is there any possible function in which we receive the annotations to the parent component. Say we link each annotation to a button and on click we pass data to the component and it will go to the specific page and highlight specific annotation or is this not possible? Thanks in advance

**Comment by hrynko** [comment]
Hi @dougieled, I'm not sure I got your idea. Could you maybe provide more details?

**Comment by dougieled** [comment]
So within a PDF some have certain annotations. Im just wondering are these accessible within the code? As i can see within the annotation layer that they have ids for example "data-annotation-id="358R" is this accessible through the code or how is this created?

Also noticed that in some instances of say annotating a paragraph it highlights yellow but covers over the text and unable to view the text. 

Thanks in advance for the reply

**Comment by dougieled** [comment]
Here is a bit of an example on the text getting blocked out. Seems when i annotate a pdf say by draging a line/paragraph it hides the text but selected each word by word annotating works. Is this a possible bug?
<img width="741" alt="Screenshot 2022-08-15 at 15 43 51" src="https://user-images.githubusercontent.com/28635506/184657653-a2234d94-155a-4e66-bdc3-a6ed632da9cd.png">
